classes/Member.php

```diff
@@ -65,6 +65,50 @@ public function isDead() {
 
     }
 
+    /*
+     * Determine if the member is a new member of a house where new is
+     * within the last 6 months.
+     *
+     * @param int house - identifier for the house, defaults to 1 for westminster
+     *
+     * @return boolean
+     */
+
+    public function isNew($house = 1) {
+        $entered_house = $this->entered_house();
+
+        if ( $entered_house && isset($entered_house[$house]) ) {
+            $date_entered = new \DateTime($entered_house[$house]['date']);
+            $now = new \DateTime();
+
+            $diff = $date_entered->diff($now);
+            if ( $diff->y == 0 && $diff->m <= 6 ) {
+                return TRUE;
+            }
+        }
+
+        return FALSE;
+    }
+
+    /*
+     * Get the date the person first entered a house. May not be for their current seat.
+     *
+     * @param int house - identifier for the house, defaults to 1 for westminster
+     *
+     * @return string - blank if no entry date for that house otherwise in YYYY-MM-DD format
+     */
+
+    public function getEntryDate($house = 1) {
+        $date_entered = '';
+
+        $entered_house = $this->entered_house();
```

Comment:
This could be `$this->entered_house($house)` which then returns either $entered_house[$house] or null straight off.